.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,20 +1,20 @@
 ---
 name: Bug Report
 about: Bug reports about the Solidity Compiler.
+labels: ["bug :bug:"]
```

Comment:
Currently (at least that's how I have handled it), the label served the purpose of "confirmed bug". What do we want to do about that? Just keep the label and use the project columns to track confirmed / not confirmed?

Reply:
Yeah, I had doubts about this myself because of that system. Maybe we should actually change it? I think it would be useful to have this stuff immediately categorized and a `confirmed` label would have more obvious meaning - the current system is a bit too implicit.

But I can remove it from the template if you prefer to stay with the current system.

Reply:
let's use the columns in the project to mark it as confirmed.

Reply:
ok, so not in `New issues` means that it's confirmed?

Reply:
yes